.github/Pull_Request_and_Code_Review_Standards.md

```diff
@@ -24,6 +24,8 @@
 
 -   If it's a PR that adds significant new functionality, there might be multiple tasks to be completed. In such cases, try to prepare a checklist in the PR description and keep it updated so that the reviewers and contributors always have an up-to-date view of the status of the PR.
 
+-   Any PR that makes a user-visible change should include a bump to package version number and entry in the `NEWS.md` (with the name of the contributor if the contributor is not an author and issue number if available).
```

Comment:
We should give some more guidance on the numbering here. Should they bump the 4th digit (e.g., `x.x.x.9001`)?

Reply:
We don't consistently use `.9000` idiom, so I have included a more general version of this.